changelog.txt

```diff
@@ -3,3 +3,4 @@
 * Fixes bug in Firestore emulator where property paths with special characters would cause errors due to ClassNotFound exceptions.
 * Fixes bug in Firestore emulator where auto-id allocation only worked once per collection.
 * Adds REST API to Firestore emulator for setting security rules.
+* Functions emulator now limits request sizes to 10mb.
```

Comment:
Okay, maybe I'm being pedantic, but wouldn't `mb` in this context read "mili-bits"? I understand that the parser options may need to be lowercase, but since this is human readable we should get the units right. So, looking at the documentation, this should be `to 10MB`?

Reply:
I though the same when writing it. Pushed a change to uppercase it.